### PR TITLE
LPS-99339 Standardize Ajax requests in frontend-js-aui-web (part2)

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test

**Comment by julien** [comment]
@carloslancha I thought I'd send this one to you for a first review (when you can).
Thanks!

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/poller.js [hunk 1]
```diff
@@ -161,15 +161,26 @@ AUI.add(
 
 				var requestStr = JSON.stringify([_metaData]);
 
-				A.io(_getReceiveUrl(), {
-					data: {
+				const data = new URLSearchParams();
+				data.append('pollerRequest', requestStr);
+
+				Liferay.Util.fetch(_getReceiveUrl(), {
+					body: JSON.stringify({
```
Comment (carloslancha) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/poller.js]: duplicated body

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/poller.js [hunk 2]
```diff
@@ -161,15 +161,26 @@ AUI.add(
...
 						pollerRequest: requestStr
+					}),
+					body: data,
+					headers: {
+						'Content-Type':
+							'application/x-www-form-urlencoded; charset=UTF-8'
```
Comment (carloslancha) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/poller.js]: Not needed

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/poller.js [hunk 3]
```diff
@@ -201,15 +212,21 @@ AUI.add(
...
-				A.io(_getSendUrl(), {
-					data: {
-						pollerRequest: requestStr
+				const data = new URLSearchParams();
+				data.append('pollerRequest', requestStr);
+
+				Liferay.Util.fetch(_getSendUrl(), {
+					body: data,
+					headers: {
+						'Content-Type':
```
Comment (carloslancha) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/poller.js]: Not needed

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/store.js [hunk 4]
```diff
@@ -127,10 +107,50 @@ AUI.add(
...
+				});
+
+				Liferay.Util.fetch(
 					themeDisplay.getPathMain() + '/portal/session_click',
-					config
-				);
+					{
+						body: data,
+						headers: {
+							'Content-Type':
```
Comment (carloslancha) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/store.js]: Not needed

**Comment by carloslancha** [comment]
LGTM

**Pull request closed without merging**